engines/python/setup/djl_python/scheduler/seq_batch_scheduler.py

```diff
@@ -55,7 +55,8 @@ def add_request(self,
                     search_algorithm: str = None,
                     search_configs: List[SearchConfig] = None,
                     kv_cache: Union[Tuple, None] = None,
-                    kv_cache_prompt_ids: Union[Dict[int, torch.tensor], None] = None):
+                    kv_cache_prompt_ids: Union[Dict[int, torch.tensor],
```

Comment:
In future, please create a PR that only do the format. It will be easy for others to review the code. It's not clear what exactly this PR is doing